src/webui/webui.h

```diff
@@ -53,7 +54,13 @@ class WebUI final : public ApplicationComponent<QObject>
 public:
     explicit WebUI(IApplication *app);
 
+    bool isEnabled() const;
     bool isErrored() const;
+    bool isHttps() const;
+    QHostAddress hostAddress() const;
+    quint16 port() const;
+    QString username() const;
+    QByteArray password() const;
```

Comment:
These two getters doesn't seem necessary IMO. Since other code get/set the value from Preferences directly, Preferences should be the ground truth. Having those could cause confusion.